1-ES6-practice/diego-ramos/starter-code/js/app.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+
+const names = ['bag', 'banana', 'bathroom', 'boots', 'breakfast', 'bubblegum', 'chair', 'cthulhu', 'dog-duck', 'dragon', 'pen', 'pet-sweep', 'scissors', 'shark', 'sweep', 'tauntaun', 'unicorn', 'usb', 'water-can', 'wine-glass'];
+
+let allProducts = [];
+const container = document.getElementById('image_container');
+const viewed = [];
+const labels = [];
+const pics = [document.getElementById('left'),
+                document.getElementById('center'), //eslint-disable-line
+                document.getElementById('right')]; //eslint-disable-line
+const list = document.getElementById('productlist');
+let totalClicks = 0;
+const views = [];
+const votes = [];
+
+function Product(name) {
+  this.name = name;
+  this.path = `img/${name}.jpg`;
+  this.votes = 0;
+  this.views = 0;
+  allProducts.push(this);
+}
+
+function makeRandom() {
+  return Math.floor(Math.random() * names.length);
+}
+
+function displayPics(){
+  // roll for three random indexes
+  while(viewed.length < 6){
+    const rando = makeRandom();
+    while(!viewed.includes(rando)){
+      viewed.push(rando);
+    }
+  }
+  // TODO: In a sentence or two, explain why the previous line of code threw an error when we changed the constiable declaration from `const to `const`.
+  // the const here is out of scope. it's contained in the while loop code block, so when you try to log it, it doesn't exist anymomre (within the scope.)
+  console.log(viewed)
+
```

Comment:
Linter not passing because of missing semi-colon. I'm not taking any points but please keep in mind for the future your linter needs to have no errors.